packages/token-search-discovery-controller/CHANGELOG.md

```diff
@@ -7,14 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0]
+
 ### Added
 
-- All param types (e.g. `TokenSearchParams`, `TrendingTokensParams`, etc.) inherit from `ParamsBase`
-- Add eponymous methods to the `TokenDiscoveryApiService`
-  - Add `getTopGainersByChains`
-  - Add `getTopLosersByChains`
-- Add swappable token search to the `TokenDiscoveryApiService` ([#5640](https://github.com/MetaMask/core/pull/5640))
-- Add support for blue-chip endpoint ([#5588](https://github.com/MetaMask/core/pull/5588))
+- feat: Add swappable token search to the `TokenDiscoveryApiService` ([#5640](https://github.com/MetaMask/core/pull/5640))
+- feat: Add support for blue-chip endpoint ([#5588](https://github.com/MetaMask/core/pull/5588))
+- feat: Add top gainers losers to token discovery service ([#5309](https://github.com/MetaMask/core/pull/5309))
+- feat: add `signEip7702Authorization` to `KeyringController` ([#5301](https://github.com/MetaMask/core/pull/5301))
```

Comment:
```suggestion
- Add swappable token search to the `TokenDiscoveryApiService` ([#5640](https://github.com/MetaMask/core/pull/5640))
- Add support for blue-chip endpoint ([#5588](https://github.com/MetaMask/core/pull/5588))
- Add eponymous methods to the `TokenDiscoveryApiService` ([#5309](https://github.com/MetaMask/core/pull/5309))
  - Add `getTopGainersByChains`
  - Add `getTopLosersByChains` 
- Bump `@metamask/utils` from `^11.1.0` to `^11.2.0` ([#5301](https://github.com/MetaMask/core/pull/5301))

Reply:
Thanks @cryptodev-2s ! The `TokenDiscoveryApiService` is internal to the package, so I didn't put its changes in the changelog. I'll remove the `feat`s and add the package bump.